ui/helpers/constants/design-system.ts

```diff
@@ -35,7 +35,6 @@ export enum Color {
   infoDefault = 'info-default',
   infoMuted = 'info-muted',
   infoInverse = 'info-inverse',
-  infoDisabled = 'info-disabled',
```

Comment:
Must have been a bad merge but `infoDisabled` no longer exists in the codebase

![Screenshot 2024-06-07 at 4 03 35 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/185ecb65-356a-4b5f-af86-e84832c0facf)